src/transformers/models/seaformer/modeling_seaformer.py

```diff
@@ -0,0 +1,764 @@
+# coding=utf-8
+# Copyright 2023 NVIDIA The HuggingFace Inc. team. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+""" PyTorch Seaformer model."""
+
+
+import math
+from typing import Optional, Tuple, Union
+
+import torch
+import torch.utils.checkpoint
+from torch import nn
+import torch.nn.functional as F
+from torch.nn import BCEWithLogitsLoss, CrossEntropyLoss, MSELoss
+
+from ...activations import ACT2FN
+from ...modeling_outputs import BaseModelOutput, ImageClassifierOutput, SemanticSegmenterOutput
+from ...modeling_utils import PreTrainedModel
+from ...pytorch_utils import find_pruneable_heads_and_indices, prune_linear_layer
+from ...utils import (
+    add_code_sample_docstrings,
+    add_start_docstrings,
+    add_start_docstrings_to_model_forward,
+    logging,
+    replace_return_docstrings,
+)
+from .configuration_seaformer import SeaformerConfig
+
+
+logger = logging.get_logger(__name__)
+
+
+# General docstring
+_CONFIG_FOR_DOC = "SeaformerConfig"
+
+# Base docstring
+_CHECKPOINT_FOR_DOC = "seaformer-large"
+_EXPECTED_OUTPUT_SHAPE = [1, 128, 64, 64]
+
+# Image classification docstring
+_IMAGE_CLASS_CHECKPOINT = "seaformer-large"
+_IMAGE_CLASS_EXPECTED_OUTPUT = "tabby, tabby cat"
+
+SEAFORMER_PRETRAINED_MODEL_ARCHIVE_LIST = [
+    "seaformer-large",
+    # See all Seaformer models at https://huggingface.co/models?filter=seaformer
+]
+
+def _make_divisible(v, divisor, min_value=None):
```

Comment:
Could you replace `v` with a more descriptive variable name?
```suggestion
def _make_divisible(value, divisor, min_value=None):
```